transactions/dkg/scripts/get_thresholds.cdc

```diff
@@ -0,0 +1,17 @@
+import FlowDKG from 0xDKGADDRESS
+
+pub struct Thresholds {
+    pub let native: UInt64
+    pub let safe: UInt64
+    pub let safePercentage: UFix64
+
+    init() {
+        self.native = FlowDKG.getNativeSuccessThreshold()
+        self.safe = FlowDKG.getSafeSuccessThreshold()
+        self.safePercentage = FlowDKG.getSafeThresholdPercentage() ?? 0.5
```

Comment:
I suggest to initialize this to `0`, which makes the overall threshold be the native one.
```suggestion
        self.safePercentage = FlowDKG.getSafeThresholdPercentage() ?? 0
```

Reply:
Updated